<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface</class>
 <widget class="QMainWindow" name="interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>interface</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>741</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabBasic">
     <attribute name="title">
      <string>Basic</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Serial Comm</string>
      </property>
      <widget class="QComboBox" name="cmbPort">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>79</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM25</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>38400-8-n-1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOpen">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMonitor">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Monitor</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>691</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Communication Data</string>
      </property>
      <widget class="QPlainTextEdit" name="txtCommData">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>651</width>
         <height>41</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>521</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>TPS Calibration and Injection Time</string>
      </property>
      <widget class="QPushButton" name="btnTPSFullGet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Full</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTPSOffGet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTPSSet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnTPSGet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtOpenMs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtBaseMs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>57</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Open ms</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>57</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Base ms</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnInjecSet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnInjecGet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>50</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtIAC">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>20</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTPSOff">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>IAC</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnIACUp">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>50</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnIACDown">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>50</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Down</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtTPSFull">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="lblLogoITS">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>691</width>
        <height>331</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="gambar.qrc">:/logo/logo-its-biru-transparan.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTPSRPM">
     <attribute name="title">
      <string>TPS RPM</string>
     </attribute>
     <widget class="QTableWidget" name="tblInj">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>701</width>
        <height>241</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>12</number>
      </property>
      <property name="columnCount">
       <number>12</number>
      </property>
      <row>
       <property name="text">
        <string>0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>40</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>50</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>60</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>70</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>80</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>90</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>100</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>750</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1250</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>6000</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Injection</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tblIgn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>701</width>
        <height>241</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>12</number>
      </property>
      <property name="columnCount">
       <number>12</number>
      </property>
      <row>
       <property name="text">
        <string>0</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>30</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>40</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>50</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>60</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>70</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>80</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>90</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>100</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>750</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1250</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>1500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>2500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>3000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>4000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5000</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>5500</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>6000</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Ignition</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnInjSet">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnInjGet">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIgnSet">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>290</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnIgnGet">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="actionDefault"/>
   <addaction name="actionInfoQt"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionInfoQt">
   <property name="text">
    <string>InfoQt</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gambar.qrc"/>
 </resources>
 <connections/>
</ui>
